### PR TITLE
chore: add api-logging codeowners

### PR DESCRIPTION
**Pull request by simonz130**
- Debranded logging and error reporting from "Stackdriver"
- Added api-logging team to codeowners of logging and error reporting

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -9,3 +9,5 @@
 /pubsub/            @googleapis/api-pubsub
 /spanner/           @skuruppu
 /storage/           @tritone
+/errorreporting/    @googleapis/api-logging @nicoleczhu
```
Comment (codyoss) [inline .github/CODEOWNERS:12]: nit: remove nicoleczhu from both as she is included in the group.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -9,3 +9,5 @@
...
+/logging/           @googleapis/api-logging @nicoleczhu
```
Comment (codyoss) [inline .github/CODEOWNERS:13]: nit: `\n`

**Review by codyoss — COMMENTED** [review]
LGTM, just a couple nits

File: .github/CODEOWNERS (hunk 1 above)
Comment (simonz130) [reply .github/CODEOWNERS]: done

File: .github/CODEOWNERS (hunk 2 above)
Comment (simonz130) [reply .github/CODEOWNERS]: done

**Review by codyoss — APPROVED** [review]

**Pull request merged**